rlclientlib/schema/v2/Event.fbs

```diff
@@ -9,12 +9,15 @@ table Event {
 }
 
 table BatchMetadata {
-    content_encoding: string;
+    content_encoding: string; //valid values: IDENTITY and DEDUP
+}
+
+table SerializedEvent {
+    payload:[ubyte]; //serialized Event objects
 }
 
-// Collection of Ranking events
 table EventBatch {
-    events:[Event];
+    events:[SerializedEvent];
     metadata: BatchMetadata;
 }
```

Comment:
Looking at the schema we would have to deserialize the payload in `SerializedEvent` to get the event-id, right? Would it be worth adding the event id as a field to `SerializedEvent`? Could that save us from having to deserialize the event to retrieve the event id?